helm/cluster-vsphere/files/etc/containerd/config.toml

```diff
@@ -6,7 +6,7 @@ subreaper = true
 # set containerd's OOM score
 oom_score = -999
 disabled_plugins = []
-[plugins."containerd.runtime.v1.linux"]
+[plugins."io.containerd.runtime.v1.linux"]
```

Comment:
Containerd failed because of the wrong plugin here. I guess it was already not working and there is a new check in containerd's new versions in the latest flatcar images.